pymc3/variational/advi.py

```diff
@@ -222,7 +216,7 @@ def run_adagrad(uw, grad, elbo, n, learning_rate=.001, epsilon=.1, verbose=1):
 
 def variational_gradient_estimate(
     vars, model, minibatch_RVs=[], minibatch_tensors=[], total_size=None, 
-    n_mcsamples=1):
+    n_mcsamples=1, seed=1):
```

Comment:
As in scikit-learn, if None is given, seed is initialized based on current state of the machine. The random number generator defined in Theano does not accept None as an argument, so add lines to make the seed random.